lessons/13-server-rendering/README.md

```diff
@@ -27,21 +27,17 @@ var path = require('path')
 
 module.exports = {
 
-  entry: path.resolve(__dirname, 'server.js'),
+  entry: './server.js',
 
   output: {
-    filename: 'server.bundle.js'
+    filename: 'server.bundle.js',
+    libraryTarget: 'commonjs2'
```

Comment:
The Webpack documentation on this property isn't very thorough. However, as mentioned above, setting `output.libraryTarget` will tell "webpack to resolve external dependencies as commonjs".

This was already being done on an individual basis in the previous `externals` property.